code/index.html

```diff
@@ -3,20 +3,117 @@
   <head>
     <meta charset="utf-8" />
     <title>Project 2: The News Site</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <!--external CSS connection-->
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <!-- meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
   </head>
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
+<!--menu-->
+<header class="mobilemenu">
+	<div class="dropdownlogo" style="float:left;">
+      	<img class="item1" src="img/logo.jpg" alt="logo"/>
```

Comment:
Look over how you use indentation and spacing for the `img`tag. 
![Example - img-tag](https://user-images.githubusercontent.com/54495430/151027688-52d23547-d036-4c57-b22a-2988e7d14370.png)